designs/2019-esm-compatibilty/README.md

```diff
@@ -0,0 +1,222 @@
+- Start Date: 2019-10-05
+- RFC PR: [#12321](https://github.com/eslint/eslint/pull/12321)
+- Authors: Evan Plaice ([@evanplaice](https://github.com/evanplaice))
+
+# ES Module Compatibility
+
+## Summary
+
+ES module support has been rolled out in all evergreen browsers and will soon land (unflagged) in node. Now is a good time to fill in the gaps to ensure ESLint users can effectively lint their ESM-based packages.
+
+## Motivation
+
+The new ES module standard is finally rolling out but, there already exists a massive ecosystem of packages build as CommonJS modules (incl ESLint).
+
+In the majority case, changing a package to be interpreted as an ES module by default is as simple as adding `"type": "module"` to `package.json`.
+
+In excepitonal edge cases it may require extra work to establish interoperability across package boundaries. The way ESLint uses `.js` config files is one of those edge cases.
+
+The goal of this RFC is to explore the different options for addressing this issue.
+
+## Detailed Design
+
+To understand the design outlined here requires some background info into the mechanics by which Node modules are loaded.
```

Comment:
I think better if this RFC describes what it changes at first of this section because it's the main stuff of this section.

Reply:
I rewrite the intro to have more substance.